compose.yml

```diff
@@ -32,13 +31,13 @@ services:
             - MW_RECAPTCHA_SITE_KEY
             - MW_RECAPTCHA_SECRET_KEY
             - MW_ADMIN_USER
-            - MW_ADMIN_PASS
+            - MW_ADMIN_PASS_FILE=/run/secrets/mw_admin_pass
             - MW_DB_NAME
             - MW_DB_INSTALLDB_USER=root
-            - MW_DB_INSTALLDB_PASS
+            - MW_DB_INSTALLDB_PASS_FILE=/run/secrets/db_root_password
```

Comment:
Same here, we don't have `MW_DB_INSTALLDB_PASS_FILE` env

Reply:
but it will work

Reply:
Same as for https://github.com/WikiWorks/BugSigDB/pull/51#discussion_r1883931412 , let's remove the ENV then